azurelinuxagent/common/utils/shellutil.py

```diff
@@ -110,6 +113,37 @@ def run_get_output(cmd, chk_err=True, log_cmd=True, expected_errors=[]):
     return 0, output
 
 
+def _encode_command_output(output):
+    return ustr(output, encoding='utf-8', errors="backslashreplace")
+
+
+def run_command(command):
+    """
+    Wrapper for subprocess.Popen. Executes the given command and returns its stdout.
+    The command parameter is a list of strings, e.g. ["ps" "aux"].
```

Comment:
I'd remove this: 

> The command parameter is a list of strings

It could also be a string (e.g. a command with no arguments)

Reply:
Thanks, removed.